src/assets/scss/App.scss

```diff
@@ -113,3 +113,21 @@
 button {
   outline: none;
 }
+.overlay {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, .2);
```

Comment:
Since we are using sass, could we have predefined colors and re-use them throughout the project?